versions.lock

```diff
@@ -0,0 +1,44 @@
+# Run ./gradlew --write-locks to regenerate this file
+com.github.jnr:jffi:1.2.17 (1 constraints: 8b0ce430)
+com.github.jnr:jnr-a64asm:1.0.0 (1 constraints: e709bdb2)
+com.github.jnr:jnr-constants:0.9.11 (3 constraints: 3c235691)
+com.github.jnr:jnr-enxio:0.20 (1 constraints: d20cde08)
+com.github.jnr:jnr-ffi:2.1.9 (3 constraints: d722dd54)
+com.github.jnr:jnr-posix:3.0.47 (1 constraints: 3c0d2523)
+com.github.jnr:jnr-unixsocket:0.22 (1 constraints: d804f130)
+com.github.jnr:jnr-x86asm:1.0.2 (1 constraints: e909bfb2)
+com.google.android:android:4.1.1.4 (1 constraints: 6a05a340)
+com.google.code.findbugs:jsr305:3.0.2 (3 constraints: c81702a8)
+com.google.errorprone:error_prone_annotations:2.2.0 (1 constraints: 160aebb4)
+com.google.guava:failureaccess:1.0.1 (1 constraints: 140ae1b4)
+com.google.guava:guava:27.0.1-jre (2 constraints: 250f4b81)
```

Comment:
Running `./gradlew why --hash 250f4b81` explains what's going on here

```
> Task :why
com.google.guava:guava:27.0.1-jre
        projects -> 27.0.1-jre
        io.airlift:airline -> 21.0
```

If a new airline version happened to pull in a newer version of guava, Gradle would pick the newer version for you and you'd see the change reflected here.